api/report_server.thrift

```diff
@@ -170,18 +177,18 @@ service codeCheckerDBAccess {
                        1: i64 reportId)
                        throws (1: shared.RequestFailed requestError),
 
-  // get the results for one runId
+  // get the results for some runIds
   ReportDataList getRunResults(
-                               1: i64 runId,
+                               1: list<i64> runIds,
```

Comment:
E.g.: User wants to filter by severity level (UNSPECIFIED = 0,STYLE = 1) and review status (FALSE_POSITIVE = 2, WONT_FIX = 3).
- severity levels come together in an `OR` clause. (0 `OR` 1)
- review statuses come together in an `OR` clause too. (2 `OR` 3)
- fields are come together in an `AND` clause.

So the filter in this case will be: `(0 OR 1) AND (2 OR 3)`